integration/setup/setup.go

```diff
@@ -56,17 +58,32 @@ func installResources(ctx context.Context, config Config) error {
 		}
 	}
 
-	var operatorTarballPath string
+	var operatorTarballURL string
 	{
-		config.Logger.Debugf(ctx, "getting tarball URL")
+		config.Logger.Debugf(ctx, "getting %#q tarball URL", project.Name())
+
+		o := func() error {
+			operatorTarballURL, err = appcatalog.GetLatestChart(ctx, key.ControlPlaneTestCatalogStorageURL(), project.Name(), env.CircleSHA())
+			if err != nil {
+				return microerror.Mask(err)
+			}
+
+			return nil
+		}
```

Comment:
A retry is needed here too. As we can't install app-operator as an app in the tests.